W/WrapIt/build_tarballs.jl

```diff
@@ -0,0 +1,111 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "WrapIt"
+version = v"1.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/grasph/wrapit.git", "46b250d663b07a54c6438745aa4aa6f6bc05ac03")
+    ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
+                  "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+# Default binarybuilder darwin chaintools miss support for filesytem
+# Install a newer SDK which supports following the recipee from
+# https://github.com/JuliaPackaging/Yggdrasil/pull/2741
+if [[ "${target}" == x86_64-apple-darwin* ]]; then
+    pushd $WORKSPACE/srcdir/MacOSX10.*.sdk
+    rm -rf /opt/${target}/${target}/sys-root/System
+    cp -ra usr/* "/opt/${target}/${target}/sys-root/usr/."
+    cp -ra System "/opt/${target}/${target}/sys-root/."
+    export MACOSX_DEPLOYMENT_TARGET=10.15
+    popd
+fi
+
+######################################################################
+# Build the software
+clang_resource_dir=clang/res
```

Comment:
I think these directory is a broken symlink in the resulting tarball.

Reply:
It is intended. The link points to a directory from the Clang@16 artifact that needs to be installed alongside. This _jll package is expected to be used through the [WrapIt](https://github.com/grasph/WrapIt.jl) package (whose version number will match with the _jll one). 

I would have preferred to put the dependency in the Artifacts.toml of the _jll package, but  I have not found how to make it happens.

Reply:
> The link points to a directory from the Clang@16 artifact that needs to be installed alongside.

Uhm, that's not how this works.  Artifacts can be installed in different depots, we can't guarantee at all that the desired Clang_jll package will be where you expect it, actually, that's also a wrong expectation which since Clang_jll isn't even a runtime dependency.

Reply:
I hadn't consider the case of multiple depot. 

So I we to copy the files instead of making link. The files to copy represent an overhead on the artifact size of 2.8% only (*). Therefore, as we already copy the libraries which are the biggest pieces, we don't save much disk space with this link and a copy is fine. 

(*) computed for x86_64 with Linux, glibc, and cxx11 string ABI.